Controller/Quiz.controller.js

```diff
@@ -0,0 +1,115 @@
+const quizdb=require('../Models/quiz.model')
+const admindb=require('../Models/admin.model')
```

Comment:
index.js can be created to import models

Reply:
Sure I'll refactor